src/Time.hpp

```diff
@@ -113,22 +121,29 @@ namespace base
         * Create a time object from an input string, interpreted as local time
         *
         * \param stringTime String describing the time
+        *                   If the string ends with the offset to UTC '%z' (\see
+        *                   documentation of strftime), this offset is applied to
+        *                   return a UTC based time object.
+        *                   If the string omits the '%z' suffix it is interpreted as a local time.
         * \param resolution Set to a resolution higher than Secs if a (non-standard)
         *                   msec or usec field is present. The milliseconds or
         *                   microseconds are added after the time string, separated by
         *                   a colon, e.g. 2012-06-14--12.05.06Z:00100
-        * \param mainFormat valid format for strptime, e.g."%Y%m%d-%H:%M:%S". Note
-        *                   that the timezone field is parsed but basically ignored,
-        *                   i.e. the string is always interpreted as a local time.
+        * \param mainFormat valid main format for strptime, e.g."%Y%m%d-%H:%M:%S". Note
+        *                   that the timezone field %Z is parsed but basically
+        *                   ignored.
         *                   The default format converts the value returned by
         *                   fromString (i.e. Time::fromString(time.toString()) is
         *                   the identity).
+        * \returns a Time object representing UTC if the input string is
+        *           suffixed by the offset to UTC '%z', e.g. +0230 for two hours
+        *           and 30 minutes (east of GMT), and localtime otherwise
```

Comment:
not 'localtime', 'the time represented by the string'. Which could itself be UTC.

Reply:
IMO "the time represented by the string" misses out that the string is interpreted for the current timezone. If the current timezone is GMT then localtime == UTC?

Reply:
Yes, you are right. Which means that the Time object represents ... whatever.

That is "the time from the string offset with the current time zone offset".

Maybe a better description would be 

~~~
If the input string contains a time offset representation of the form +HHMM (+0230 for 2:30 east of UTC) and the format captures it with `%z`, the value returned is the UTC time. Otherwise, the string is interpreted as a time in the local time zone and converted to UTC accordingly.
~~~

Reply:
Updated with slight changes to point out that right now only the +HHMM at the end of the string is supported.